gap/PolygonalComplexes/animating.gd

```diff
@@ -0,0 +1,537 @@
+#! @Returns the updated print record.
+#! @Arguments surface, i, colour, printRecord
+DeclareOperation( "SetEdgeColour", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsString, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "GetEdgeColour", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @EndGroup
+
+
+#! @BeginGroup SetFaceColours
+#! @Description
+#! Set colours for all faces. (Colours are stored in the format 0xABCDEF where A,B,C,D,E,F are elements of the hexadecimal code.  For more information look at the start of the section <Ref Subsect="Section_LabelColouring"/>).
+#! @Returns the updated print record.
+#! @Arguments surface, newColoursList, printRecord
+DeclareOperation( "SetFaceColours", [IsSimplicialSurface and IsNotEdgeRamified, IsDenseList, IsRecord] );
+#! @Arguments surface, printRecord
+DeclareOperation( "GetFaceColours", [IsSimplicialSurface and IsNotEdgeRamified, IsRecord] );
+#! @EndGroup
+
+#! @BeginGroup SetFaceColour
+#! @Description
+#! Set the <A>colour</A> of the face in position <A>i</A> of <A>Faces(surface)</A>. (Colours are stored in the format 0xABCDEF where A,B,C,D,E,F are elements of the hexadecimal code.  For more information look at the start of the section <Ref Subsect="Section_LabelColouring"/>).
+#! @Returns the updated print record.
+#! @Arguments surface, i, colour, printRecord
+DeclareOperation( "SetFaceColour", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsString, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "GetFaceColour", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @EndGroup
+
+#! @Section Inner Circles
+#! @SectionLabel LabelInnerCirclesAnimating
+#!
+
+#! For an inner circle we need a face with 3 corresponding edges. Then an inner circle is an circle which lie in the face such that the circle touches each edge of the face in exactly one point.
+#!
+#! The functionality concerning the inner circles is completely simular to the functionality converning the edges.
+#!
+#! By default the inner circles are inactive.
+#!
+#! The following options are implemented for inner circles:
+#! <Enum>
+#! <Item>Calculate the parameters of inner circles by the 3D-coordinates of the vertices. Note that this is necessary for the animation. Therefore the function <K>CalculateParametersOfInnerCircle</K> gets called after using the function <K>DrawSurfaceToJavaScript</K> if no parameters are set for the inner circles. If parameters are set for the inner circles and the 3D-coordinates of a vertex has been changed, then you have to call the function <K>CalculateParametersOfInnerCircle</K> manuel again if you use the function <K>DrawSurfaceToJavaScriptCalculate</K> with <A>calculate = false</A>. Otherwise the animation will be not necessarly correct. If you use the function <K>DrawSurfaceToJavaScript</K> the parameters will be updated automatically. </Item>
```

Comment:
This is a daunting wall of text. I would put this as the third bullet point (since it is less important) and phrase it like this:

If the manual-calculation mode from Section [REF] is used, inner circles (like edges) have to be manually recomputed. The method is ... (REF). Compare the description in Section REF for more details on this mode.